services/personal/api.go

```diff
@@ -62,7 +62,7 @@ func (api *PublicAPI) SetRPC(rpcClient *rpc.Client, timeout time.Duration) {
 	api.rpcTimeout = timeout
 }
 
-// Recover is an implementation of `personal_ecRecover` or `web3.personal.ecRecover` API
+// Recover is an implementation of `personal_ecRecover` or `web3.eth.personal_ecRecover` API
```

Comment:
if I start a `geth` console I see it as you defined it before:

```
> web3.personal.ecRecover
function()
```

Reply:
yeah, that is our `web3.js` vs how MetaMask shows it in their docs.
In reality, people are using something like `sendAsync( "personal_sign", params: [from, data])`